benchmarks/distributed/rpc/parameter_server/test_scripts.py

```diff
@@ -0,0 +1,23 @@
+import subprocess
```

Comment:
curious, do we expect this .py file or the above .sh file to be the entrance script? I'd assume it's the .sh file, as running `python test_scripts.py` won't do anything IIUC. In this case, what's the purpose of having this script?

Reply:
The entrance will be .py file for checking errors using pytest. And .sh file for running the benchmark using python. 

A user can use the script to check subsets of the experiment scripts added to the test script to see if the subset is passing. I think it will make checking the scripts easier for larger subset sizes.